### ISSUE TITLE
SA1000 False Positive for C# 9 target-typed new

### ISSUE THREAD

**Issue by pavel-mikula-sonarsource** [DotNetAnalyzers/StyleCopAnalyzers#3414]
Rule SA1000 should not fire for C# 9 target-typed new keyword followed by an open brace:
```
public object NewObject() => new();
public System.Exception NewException(string message) => new(message);
```
Visual Studio default formatting doesn't put a space there.

**Comment by pavel-mikula-sonarsource** [comment]
@sharwell can you please check that this makes sense? If so, I can give it a try to create a PR to fix it.

**Comment by sharwell** [comment]
Duplicate of #3214
